### PR TITLE
Vikas/dependebot

### PR DESCRIPTION
**Pull request by vikas0212git**
## iotedgedev Pull Request Checklist

- [x] Versions updated (both __init__.py files, CHANGELOG, setup.py, setup.cfg)
- [x] Unit tests pass locally
- [ ] Quickstart steps locally validated
- [x] Passes unit tests in CICD pipeline (green on Github pipeline)
- [ ] Pypi RC version passes Edge CICD pipeline validation for cross-tool validation

## Additional comments

### CODE REVIEW

**Comment by marianan** [comment]
could you also increment the iotedgedev version in setup.py and init.py?

File: requirements_dev.txt [hunk 1]
```diff
@@ -8,9 +8,9 @@ pycodestyle==2.7.0
 autopep8==1.5.7
 tox==3.24.1
 coverage==4.1
-Sphinx==1.4.8
+Sphinx==4.4.0
 cookiecutter==1.7.3
-PyYAML==4.2b4
+PyYAML==5.4
```
Comment (mhshami01) [inline requirements_dev.txt:13]: should this be >=?

Comment (vikas0212git) [reply requirements_dev.txt:13]: changed

**Review by marianan — APPROVED** [review]
:shipit:

**Pull request merged**